openwisp_radius/api/views.py

```diff
@@ -466,8 +524,8 @@ def post(self, request, *args, **kwargs):
         )
         serializer.is_valid(raise_exception=True)
         user = self.get_user(serializer, *args, **kwargs)
-        token, created = UserToken.objects.get_or_create(user=user)
-        radius_token = self.get_or_create_radius_token(user)
+        token, _ = UserToken.objects.get_or_create(user=user)
+        radius_token = self.get_or_create_radius_token(user, self.organization, True)
```

Comment:
two things:

1. let's pass `enable_auth=True` instead of `True`, for readability
2. we should add another flag to this method to renew the token, eg:

```python
radius_token = self.get_or_create_radius_token(user, self.organization, enable_auth=True, renew=True)
```

We should renew the token each time the user uses obtain auth token or validate auth token, this is related to `DISPOSABLE_RADIUS_USER_TOKEN`.

Please also write tests for this (1 for obtain auth token view and another one for validate auth token view).

Reply:
Just to be clear, the renew flag will delete the existing `RadiusToken` and create a new one, is that correct?

Reply:
I would just update the current one for efficiency, these methods can be used by many thousands of users multiple times a day

Reply:
The `key` attribute of `RadiusToken` is the model's primary key, and as far as I know, primary key cannot be changed without creating a new model, I think we should create a new `UUID` primary key, what do you think, that will allow us to not delete the `RadiusToken` for the change!?

Reply:
oh damn, you're right, I forgot there was no UUID on this model. Let's keep it how you did it. Thanks